Changelog.md

```diff
@@ -0,0 +1,71 @@
+# MSBuild 16.10.0
```

Comment:
do you intend to backfill the changelogs for earlier releases from https://github.com/dotnet/msbuild/releases ?

Reply:
Hm I don't think I've ever seen a project where a Changelog.md only contained the current version.

Most have all of the releases in it and a section for v.current, e.g. https://github.com/emscripten-core/emscripten/blob/main/ChangeLog.md

I'd prefer that, or at least have links to the earlier release notes in the doc.

Reply:
Sounds reasonable to me. I'll add at least the 16.9 release.